nucypher/policy/conditions/context.py

```diff
@@ -133,6 +133,32 @@ def get_context_value(
     return value
 
 
+def extract_user_address_from_context(context: Dict[str, Any]) -> ChecksumAddress:
```

Comment:
I'd remove this function and use `resolve_any_context_variables` below. See my other comment for the rationale.